Day5/que5.js

```diff
@@ -0,0 +1,23 @@
+// Write a program flattenObject that takes an object with nested properties and returns a flat object with the nested properties transformed into dot-separated properties. The function should handle nested objects and arrays.
```

Comment:
@anantmishra-successive the o/p of this ques should have a flat object with nested keys dot separated but not starting with dot. 
Secondly, you are not supposed to flatten array.

Reply:
Sir
Resolved

Reply:
![updateday5](https://github.com/anantmishra-successive/JS_Training/assets/162098832/8c1a0acc-092c-40bd-be28-30b3dc4b978d)

Reply:
@anantmishra-successive 
1.You have added extra logic. Do it by manipulating same old logic.
2. And DO NOT resolve the comments by yourself.
3. I also wrote that you are not supposed to flatten the array. You haven't yet changed that.

Reply:
![day5final](https://github.com/anantmishra-successive/JS_Training/assets/162098832/ca3717e3-0531-4665-8268-22d4ded63114)